debian/control

```diff
@@ -32,10 +32,14 @@ Description: Provisioning backend for Homestead, the Cassandra powered HSS gatew
 
 Package: homer-cassandra
 Architecture: any
-Depends: clearwater-cassandra, clearwater-infrastructure
+Depends: clearwater-cassandra, clearwater-infrastructure, crest-prov
```

Comment:
The default behavior is to install recommends. but not install suggests.

You can change this with `--no-install-recommends`, and `--install-suggests`